dbt/adapters/databricks/connections.py

```diff
@@ -475,6 +480,84 @@ class DatabricksConnectionManager(SparkConnectionManager):
     TYPE: str = "databricks"
     credentials_provider: Optional[TCredentialProvider] = None
 
+    def cancel_open(self) -> List[str]:
+        run_id_dir = f"/Shared/dbt_python_model/{self.profile.credentials.database}/{self.profile.credentials.schema}/run_ids/"
+        run_ids = self._list_files(run_id_dir)
+        print(run_ids)
+        for run_id in run_ids:
+            self._cancel_run_id(run_id_dir, run_id)
+
+        return super().cancel_open()
```

Comment:
Below you raise an exception on a non-200, but that will interrupt cancelling the other operations.  Better to log a warning on non-200 I think.